stdlib/universe/range.go

```diff
@@ -248,16 +260,13 @@ func (t *rangeTransformation) Process(id execute.DatasetID, tbl flux.Table) erro
 		// If [start, stop) (where start <= stop) from the group key is contained in the
 		// range transformation bounds [keyStart, keyStop], we can skip the whole table.
 		// Still want to skip if start >= keyStart and t.bounds.Stop == keyStop
-		forwardTable = t.bounds.Contains(keyStart) && (t.bounds.Contains(keyStop) || t.bounds.Stop == keyStop)
-	}
-
-	if forwardTable {
-		return execute.AppendTable(tbl, builder)
+		if t.bounds.Contains(keyStart) && (t.bounds.Contains(keyStop) || t.bounds.Stop == keyStop) {
```

Comment:
I'm curious. What happens if `_start` and `_stop` are lying and there is a `_time` that isn't in the range? Or there is a null time (which should be discarded) inside of a range that is otherwise valid?

Reply:
In that case, the lie would be propagated.  The alternative is to never forward tables... I can see reasons for doing it either way.  What do you think?

Reply:
So, I realized a couple things:
1. transformations can only iterate over the input tables one time.  So the logic to check if table forwarding would be allowed would need to be put into the call to `tbl.Do` at the end of the `Process` method.
2. `execute.AppendTable` just calls `tbl.Do` anyway, so there's not a whole lot of performance benefit to doing the forwarding... it's not as if O(n) becomes O(1).

As such I just removed the table-forwarding logic.

Reply:
And there is an E2E test failing after rebasing on master, I need to figure that out.

Reply:
This is something worth exploring later. We should be capable of table forwarding although I do believe you are correct about its current state so we'll revisit forwarding when it becomes  practical.